modules/002-deckhouse/templates/validation.yaml

```
@@ -0,0 +1,76 @@
+{{- if semverCompare ">= 1.26" .Values.global.discovery.kubernetesVersion }}
+{{- $policyName := "d8-create-namespace-validator" }}
```

```suggestion
{{- $policyName := "system-ns.deckhouse.io" }}
```

Let's name it it a more predictable way.
Sorry for that, but it's a bit closer to [docs](https://kubernetes.io/docs/reference/access-authn-authz/validating-admission-policy/#creating-a-validatingadmissionpolicy) and looks really more readable, because we will have an error like:

```
ValidatingAdmissionPolicy 'system-ns.deckhouse.io' with binding 'system-ns.deckhouse.io' ...
```
which will definitely points to the deckhouse source